New package: Kinbiont v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Kinbiont
- Repository: https://github.com/pinheiroGroup/Kinbiont.jl
- Created by: @aivuk
- Version: v1.0.0
- Commit: c781db5d2ffa5fde36e32816c88d07bd8a2c2d38
- Reviewed by: @aivuk
- Reference: https://github.com/pinheiroGroup/Kinbiont.jl/commit/c781db5d2ffa5fde36e32816c88d07bd8a2c2d38#commitcomment-145257390
- Description: Ecosystem of numerical methods for microbial kinetics data analysis, from preprocessing to result interpretation.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6c29ec51dbfaaba8e46cc24dd6119623988a0691e1ef9b919f08b484e7b3617613a2d046d6b770e0b4267c2b1a32300f518ecace30ac842cab206fd53593a0196e01ea43b288a0eee73ddb9bd5d71e7a73be03deaaffb3192500c6318e1e531549bbc9447661a4e1923576191eebb2b24f96fd2e04ef51c7f0f62b01cc76c25dc94cface9402f3dedac026c67a436df3e42b27614993c84bc9db248d722951a3c953d3f1c6f833129cabd46d07bb25f0a -->

aivuk commented:
> Thank you for submitting your package! Two small concerns:
> 
>     * It seems you have a Documenter-based documentation, but I could not find a documentation link. Can you link from the README to the deployed documentation?

Thanks I added a link to the documentation website now: https://kinbiont.fuzue.org/

>     * The licensing is ambiguous: there is both a GPL license in the repo and what looks like MIT. If you intend to dual license, please make that clear in the README. I would caution against using GPL (as opposed to LGPL) for Julia packages. Given that GPL-code cannot be linked with non-GPL code, that raises questions about how a library using this package as a dependency can be distributed. MIT would be the license that will give you the least friction in integrating with the wider Julia ecosystem.

I left now only the LICENSE file and removed the COPYING.txt.

Thanks for the quick review!

aivuk commented:
I forgot to mention, I also changed to MIT license.

goerz commented:
[noblock] Looks great! You should be good to go then, as soon as you add the Julia compat bound in your Project.toml. When you do, just retrigger the registration without changing the version number. It will then update this existing PR.

You also have to edit all your comments above to contain `[noblock]`

aivuk commented:
ok, thanks. I did add the `[compat]` to Project.toml and retriggered adding a new comment one hour ago but there is no update. Do I need to do something else or just wait?  [noblock]

goerz commented:
[noblock] That's strange. It should have updated. I don't have any experience with the issue-based workflow for registration that you seem to be using. I usually comment `@JuliaRegistrator register` on the *commit* that should be tagged.

JuliaRegistrator committed:
New package: Kinbiont v1.0.0

UUID: 3b7c519f-be5e-4159-a208-6486ce96bb37
Repo: https://github.com/pinheiroGroup/Kinbiont.jl.git
Tree: aacf216eab8d5f3058e7f65da8d742acb9c77819

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
